README.md

```diff
@@ -1,17 +1,23 @@
 # Title <!-- Replace 'Title' and this comment with a title or name of your algorithm -->
-Description <!-- Replace this entire line with a short description of your algorithm -->
+The Greenness Transformer provides a series of calculations for rgb image data used in monitoring plant growth and health.
+A further explanation for the transformer can be found at [this link](https://docs.google.com/document/d/1cAm5w1Bs6dB1SHgf-HVmwwbebLhbZMUvTkXLtas_-xI/edit)
 
 ## Authors
-<!-- Replace this comment with a list of the authors of the algorithm pointing out who the main contacts are -->
+Chris Schnaufer
```

Comment:
This should contain ways of contacting the people listed. Otherwise all you have is a name - which isn't as useful as it can be. Also add yourself.